Small robotics layout change.

Birdtalon opened the pull request:
Small change to the layout of robotics to address some requests/complaints.

- Added a secure roboticist's locker containing backpacks, labcoats and headsets for the roboticist.
- Moved the diagnostic HUDs out of the closet and onto the table next to the north window.
- Removed the headset from the rack.
- Nudged the Science SOP book down to reveal the flashes on the table at round start.
- Moved all flashes into one stack.
- Removed both stools and moved the office chair and spawn point next to the north window.

![dreammaker_2017-09-14_14-36-45](https://user-images.githubusercontent.com/25027759/30433040-535c0496-995b-11e7-8567-6613d8705cf7.png)

🆑 
add: Roboticist's locker containing headsets, backpacks and coats.
tweak: Slightly re-arranged some items in robotics.
/🆑

SamHPurp commented on code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:
```diff
@@ -22,7 +22,28 @@
 
+	New()
+		..()
+		new /obj/item/weapon/storage/backpack(src)
```
Shouldn't they have science backpacks?

`new /obj/item/weapon/storage/backpack/science(src)` etc

Birdtalon replied:
Robotics loadout is standard backpacks apparently.

Kyep replied:
I would assume the fact that Robos don't get science backpacks is an oversight. I'd put science backpacks in their locker, and in their loadouts.

taukausanake replied:
Robotics is too special to be counted among the manchildren that is Science. Also those backpacks would look ugly for Roboticist. Special Robotics backpack when?

FalseIncarnate commented on _maps/map_files/cyberiad/cyberiad.dmm:
```diff
@@ -4562,8 +4562,14 @@
 "bJN" = (/obj/machinery/atmospherics/unary/vent_pump{on = 1},/obj/effect/decal/warning_stripes/north,/turf/simulated/floor/plating,/area/turret_protected/aisat_interior)
+"bJO" = (/obj/machinery/firealarm{dir = 4; pixel_x = 24},/obj/structure/rack{dir = 8; layer = 2.9},/obj/item/weapon/storage/toolbox/electrical{pixel_x = 1; pixel_y = 6},/obj/item/weapon/storage/toolbox/mechanical{pixel_x = -2; pixel_y = -1},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{dir = 4},/obj/item/clothing/head/welding{pixel_x = -3; pixel_y = 5},/obj/item/clothing/glasses/weldin ...
 "bJP" = (/obj/machinery/door/firedoor,/obj/machinery/door/airlock/medical{name = "Examination Room"; req_access_txt = "5"; req_one_access_txt = "0"},/turf/simulated/floor/plasteel{tag = "icon-whitebluefull"; icon_state = "whitebluefull"},/area/medical/exam_room)
+"bJQ" = (/obj/machinery/ai_status_display{pixel_x = 32; pixel_y = 0},/obj/structure/table,/obj/item/device/flash/synthetic,/obj/item/device/flash/synthetic,/obj/item/device/flash/synthetic,/obj/item/device/flash/synthetic,/obj/item/device/flash/synthetic,/obj/item/weapon/book/manual/sop_science{step_y = -14},/turf/simulated/floor/plasteel{icon_state = "white"},/area/assembly/robotics)
```
`step_y` should never be set. you want something like `pixel_y`

SamHPurp replied:
Good catch...

The fact that there were `step_y`'s involved, and you didn't notice anything weird.... you didn't test your edits on map ;)

It becomes reeeeeal fucking obvious somethings wrong with `step_y` edited, anywhere in the code, even just set to 0.

Birdtalon replied:
First time doing anything map related, how do I achieve the same thing with `pixel_y`?

Birdtalon replied:
Okay sorted this.

Kyep commented on _maps/map_files/cyberiad/cyberiad.dmm:
```diff
@@ -4562,8 +4562,14 @@
 "bJL" = (/obj/machinery/atmospherics/unary/vent_pump{on = 1},/turf/simulated/floor/plasteel{tag = "icon-whiteblue (EAST)"; icon_state = "whiteblue"; dir = 4},/area/medical/reception)
 "bJM" = (/obj/structure/grille,/obj/structure/window/full/shuttle,/turf/simulated/floor/plating,/area/shuttle/constructionsite)
 "bJN" = (/obj/machinery/atmospherics/unary/vent_pump{on = 1},/obj/effect/decal/warning_stripes/north,/turf/simulated/floor/plating,/area/turret_protected/aisat_interior)
+"bJO" = (/obj/machinery/firealarm{dir = 4; pixel_x = 24},/obj/structure/rack{dir = 8; layer = 2.9},/obj/item/weapon/storage/toolbox/electrical{pixel_x = 1; pixel_y = 6},/obj/item/weapon/storage/toolbox/mechanical{pixel_x = -2; pixel_y = -1},/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{dir = 4},/obj/item/clothing/head/welding{pixel_x = -3; pixel_y = 5},/obj/item/clothing/glasses/weldin ...
```
This rack has at least three items on it. The table next to the window is empty. Perhaps move one of these items to there?

Birdtalon replied:
Good suggestion, I will move the HUDs and the welding goggles there to balance things out a little.

Kyep commented on code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:
```diff
@@ -22,7 +22,28 @@
+		new /obj/item/weapon/storage/backpack(src)
+		new /obj/item/weapon/storage/backpack/satchel_norm(src)
+		new /obj/item/weapon/storage/backpack/satchel_norm(src)
+		new /obj/item/weapon/storage/backpack/duffel(src)
```
There is no such thing as a science duffelbag?

Birdtalon replied:
A science duffelbag does exist yes.

taukausanake commented:
@Kyep The wall under the windoor is intended because it's inside the Science wing. The rwalls are there to keep the Science hallway from being an access point. Even if you rwall those two down there you still have the normal walls next to the surgery corner. Sure more trouble to break through but it's still there

ProperPants commented:
I appreciate you moving the SoP book to make the flashes visible, but putting the diagnostic HUDs on the rack makes them a bit hard to find for newbies. Perhaps it would be better to start the HUDs elsewhere, such as on that table next to the window that is totally empty. Otherwise, this seems pretty good.

Birdtalon commented:
Updated.

I have incorporated some of your suggestions @Kyep with regards to the layout.
I would like to hear from a maintainer ideally if the fact that roboticists have standard backpacks and kit is indeed an oversight.

It would be nice to take this opportunity to give roboticists their own bags to match their uniforms if someone is willing to sprite it.

tigercat2000 commented:
We can change the locker contents if/when roboticists have their uniforms updated.

taukausanake commented:
What's wrong with Roboticists uniforms? I like them, they're different and special

Birdtalon commented:
The uniforms are fine, they just lack specific backpacks.

tigercat2000 commented:
@taukausanake I meant uniforms as in their uniform datums, the stuff that's equipped to them.
